enrichers/src/main/java/io/quarkus/qe/enricher/LookUpHelper.java

```diff
@@ -0,0 +1,105 @@
+package io.quarkus.qe.enricher;
+
+import org.apache.maven.model.Dependency;
+import org.apache.maven.model.DependencyManagement;
+import org.apache.maven.model.Plugin;
+import org.apache.maven.model.PluginManagement;
+
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class LookUpHelper {
```

Comment:
Can you make this helper a service a renamed it to "QuarkusVersionResolver" or similar?